.cirrus.yml

```diff
@@ -7,12 +7,12 @@ flutter_docker_builder:
     # from https://flutter.dev/docs/development/tools/sdk/releases
     matrix:
     - DOCKER_TAG: latest
-      FLUTTER_VERSION: v1.12.13+hotfix.8
+      FLUTTER_VERSION: v1.16.1
```

Comment:
Latest and stabe should be the same so please revert. 👌

Reply:
Right! Reverting now.